packages/openstellina/openstellina.0.0.1/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A http client for Stellina smart telescope by Vaonis"
+description: "Mobile Stellina App does not provide access to telescope exposures > 20s (lights or darks). This software bypasses the App and talks to the telescope directly"
+maintainer: ["Jonathan Kimmitt"]
+authors: ["Jonathan Kimmitt"]
+license: "Apache-2.0"
+tags: ["Vaonis" "Stellina" "long exposure" "GUI"]
+homepage: "https://github.com/jrrk2/openstellina.git"
+doc: "https://github.com/jrrk2/openstellina.git"
+bug-reports: "https://github.com/jrrk2/openstellina.git/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.0"}
+  "lablgtk"
+  "lwt_glib"
+  "yojson"
+  "cohttp-lwt-unix"
+  "zeit"
+  "ppx_let"
+  "quests"
+  "ppx_yojson"
+  "lwt"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/jrrk2/openstellina.git"
+url {
+  src: "https://github.com/jrrk2/openstellina/archive/refs/heads/main.zip"
```

Comment:
This points at the main branche, not at a specific release

Reply:
An egregious error. I suppose I assumed spam-publish would deal with this aspect.